pycalphad/plot/eqplot.py

```diff
@@ -137,32 +137,34 @@ def eqplot(eq, ax=None, x=None, y=None, z=None, tielines=True, **kwargs):
             two_phase_y = np.array([two_phase_y, two_phase_y]).swapaxes(0, 1)
 
         # plot two phase points
-        two_phase_plotcolors = np.array(list(map(lambda x: [colorlist[x[0]], colorlist[x[1]]], found_two_phase)), dtype='U') # from pycalphad
-        ax.scatter(two_phase_x[..., 0], two_phase_y[..., 0], s=3, c=two_phase_plotcolors[:,0], edgecolors='None', zorder=2, **kwargs)
-        ax.scatter(two_phase_x[..., 1], two_phase_y[..., 1], s=3, c=two_phase_plotcolors[:,1], edgecolors='None', zorder=2, **kwargs)
+        two_phase_plotcolors = np.array(list(map(lambda x: [colorlist[x[0]], colorlist[x[1]]], found_two_phase)), dtype='U')
+        ax.scatter(two_phase_x[..., 0], two_phase_y[..., 0], s=3, c=two_phase_plotcolors[:, 0], edgecolors='None', zorder=2, **kwargs)
+        ax.scatter(two_phase_x[..., 1], two_phase_y[..., 1], s=3, c=two_phase_plotcolors[:, 1], edgecolors='None', zorder=2, **kwargs)
 
         if tielines:
             # construct and plot tielines
             two_phase_tielines = np.array([np.concatenate((two_phase_x[..., 0][..., np.newaxis], two_phase_y[..., 0][..., np.newaxis]), axis=-1),
                                            np.concatenate((two_phase_x[..., 1][..., np.newaxis], two_phase_y[..., 1][..., np.newaxis]), axis=-1)])
             two_phase_tielines = np.rollaxis(two_phase_tielines, 1)
-            lc = mc.LineCollection(two_phase_tielines, zorder=1, colors=[0,1,0,1], linewidths=[0.5, 0.5])
+            green_tieline_color = [0, 1, 0, 1]  # RGBA
+            lc = mc.LineCollection(two_phase_tielines, zorder=1, colors=green_tieline_color, linewidths=[0.5, 0.5])
             ax.add_collection(lc)
 
     # If we found three phase regions:
-    if three_phase_idx[0].size > 0:
+    if (three_phase_idx[0].size > 0) and (len(indep_comps) == 2):
```

Comment:
This is more or less the key addition and everything else are style/naming changes, correct?

Reply:
Yes, this is the only non-cosmetic change